saleor/product/models.py

```diff
@@ -62,6 +62,9 @@ class Category(ModelWithMetadata, MPTTModel, SeoModel):
     slug = models.SlugField(max_length=255, unique=True, allow_unicode=True)
     description = SanitizedJSONField(blank=True, null=True, sanitizer=clean_editor_js)
     description_plaintext = TextField(blank=True)
+    updated_at = models.DateTimeField(
+        auto_now=True, blank=True, null=True, db_index=False
```

Comment:
Just small thing, we can remove that statement, as is redundant:
```suggestion
        auto_now=True, blank=True, null=True
```

Overall, LGTM! Thanks for the contribution!

Reply:
The field has been removed